platform-spring-bom/platform-spring-caching/src/main/kotlin/io/cloudflight/platform/spring/caching/autoconfigure/SessionAutoConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package io.cloudflight.platform.spring.caching.autoconfigure
```

Comment:
suggestion: is this still the correct package? as its not related to caching anymore, but session storage

Reply:
I agree with you, but I decided to keep it in the caching module to avoid breaking changes.